src/global/Constants.js

```diff
@@ -19,6 +19,7 @@ export const NATION_INDEX_RECOVER_PERIOD = 10 * 1000;
 
 export const TRANSACTIONS_WORKER_RECOVER_DELAY = 30 * 1000;
 
+export const PADDING_STATUS_BAR_IOS = 20;
```

Comment:
Sorry, I meant that there should already be a constant for that. The main issue with using 20 is that it's not that for iPhone X. If you want to use status bar height, please consider some way to get the correct height instead of hardcoding 20px. Btw, we have style called `statusBar` on global styles, but I don't know how exactly it's used throughout the app. It may be useful for that task as well

Reply:
fixed, pls check again